ext/fg/js/popup.js

```diff
@@ -243,6 +248,25 @@ class Popup {
         return popupPreparedPromise;
     }
 
+    _onFrameLoad() {
+        if (!(this._injectPromise !== null && this._injectPromiseComplete)) { return; }
```

Comment:
`this._injectPromise` should never be `null` when `this._injectPromiseComplete` is `true`

Reply:
Looking at it again, the `this._injectPromise` check may actually not be necessary, since `this._injectPromiseComplete` being `true` is predicated on `this._injectPromise` existing.

The way the code is used currently doesn't allow for this, but technically `this._injectPromiseComplete` could be set to `true` if `this._injectPromise` was reassigned due to anything other than that promise resolving. I may add a guard for this, since the code may potentially allow that case in the future; easier to remember now than later.

Reply:
Guard added in b13957ea8efa58bd2d88615262282fecad151f61.

Reply:
Thanks!

Reply:
Redundant check removed in ed8b438836eda6ec4532bd98c2d7b425c0870393.